atomics/T1580/T1580.yaml

```diff
@@ -0,0 +1,55 @@
+attack_technique: T1580
+display_name: 'Cloud Infrastructure Discovery'
+atomic_tests:
+- name: AWS - EC2 Enumeration from Cloud Instance
+  auto_generated_guid:
```

Comment:
Can you remove this key to fix the CI error : `auto_generated_guid`

Reply:
Done, left that in there during testing because atomic-operator apparently needs it present.